README.md

```diff
@@ -1,75 +1,17 @@
-# capability-providers
+This repository contains capability providers for wasmcloud.
```

Comment:
Please add a markdown header for this README, something like `# Capability Providers`